blobfuse/OAuthTokenCredentialManager.cpp

```diff
@@ -305,21 +349,21 @@ std::function<OAuthToken(std::shared_ptr<CurlEasyClient>)> SetUpSPNCallback(std:
         std::shared_ptr<CurlEasyRequest> request_handle = http_client->get_handle();
 
         // set up URI and headers
-        request_handle->set_url(uri_token_request_url->to_string());
-        request_handle->add_header("Content-Type", "application/x-www-form-urlencoded");
+       request_handle->set_url(uri_token_request_url->to_string());
+       request_handle->add_header("Content-Type", "application/x-www-form-urlencoded");
 
-        // Set up body query
-        auto body = std::make_shared<std::stringstream>(queryString);
-        request_handle->set_input_stream(storage_istream(body));
-        request_handle->set_input_content_length(queryString.length());
-        request_handle->add_header("Content-Length", std::to_string(queryString.length()));
+       // Set up body query
+       auto body = std::make_shared<std::stringstream>(queryString);
+       request_handle->set_input_stream(storage_istream(body));
+       request_handle->set_input_content_length(queryString.length());
+       request_handle->add_header("Content-Length", std::to_string(queryString.length()));
 
         // Set up output stream
         storage_iostream ios = storage_iostream::create_storage_stream();
-        request_handle->set_output_stream(ios.ostream());
+       request_handle->set_output_stream(ios.ostream());
 
         // Set request method
-        request_handle->set_method(http_base::http_method::post);
+       request_handle->set_method(http_base::http_method::post);

```

Comment:
NIT - The changes on lines 287 - 322, one space was removed from each line, were the indents on this part of the file wrong or could you readjust these that way it doesn't look like there were changes here.